src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Array.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
+        /// <summary>
+        ///   Reads the next data item as the start of an array (major type 4)
+        /// </summary>
+        /// <returns>
+        ///   The length of the definite-length array, or <c>null</c> if the array is indefinite-length.
```

Comment:
```suggestion
        ///   The length of the definite-length array, or <see langword="null" /> if the array is indefinite-length.
```

The docs site uses a different style for `null` (or true or false) than `<c>`).  I was going to say "so that it flips to `Nothing` when the page mode is VB", but that doesn't seem to be happening.  Perhaps they have a content generator bug.

Reply:
Changed. FWIW there are a few `<c>null</c>` occurences in Asn docs as well.

Reply:
Same for a few `<c>false</c>` and `<c>true</c>` occurrences.